packages/core/util/layouts/GranularRectLayout.ts

```diff
@@ -464,8 +454,7 @@ export default class GranularRectLayout<T> implements BaseLayout<T> {
       return
     }
 
-    const data = rect.data || rect.id
-    const { bitmap } = this
+    const data = rect.id
```

Comment:
we don't flip flop between data/id conditionally now, always return id, and the data is the stored feature (and we can get the data by inspecting the layout with the async feature details)